src/api/dataStatistics.js

```diff
@@ -1,8 +1,20 @@
 import { getInstance } from 'd2'
+import { getDataStatisticsName } from '../modules/itemTypes'
 
 export const apiPostDataStatistics = async (eventType, id) => {
     const d2 = await getInstance()
     const url = `dataStatistics?eventType=${eventType}&favorite=${id}`
 
     d2.Api.getApi().post(url)
 }
+
+// update data statistics
+export const apiPostFavoriteDataStatistics = async (id, type) => {
```

Comment:
This is almost exactly the same code as the function right above it. It should be possible to just use the existing function, and instead get the dataStatisticsName at the point the function is called (currently in Item.js)

Reply:
That would be better 😄 

I've refactored: 
https://github.com/tomzemp/dashboard-app/commit/1223f80404e0504cd7d086cfb5b649e5a1939d99